config/crd/bases/policy.networking.k8s.io_adminnetworkpolicies.yaml

```diff
@@ -88,9 +88,7 @@ spec:
                                 minimum: 1
                                 type: integer
                               protocol:
-                                allOf:
-                                - default: TCP
-                                - default: TCP
+                                default: TCP
```

Comment:
Sou manually added this right? Does it get removed when you run `make all`? I wonder why this kube builder tag isn't working right :(

Reply:
I didn't manually add it, that's the best part! not exactly sure how the `default:TCP` is populated, but it wasn't me.

Reply:
Yep theres some legacy API funniness ensuring the default, see https://github.com/kubernetes/api/blob/master/networking/v1/types.go#L148 there's no explicit default in netpol either 

This should be alright!

Reply:
It's probably that port.protocol already has TCP as its magic default somehow.